.github/workflows/release.yml

```diff
@@ -37,7 +37,7 @@ jobs:
         with:
           artifacts: "**/*.nupkg"
           token: ${{ secrets.GITHUB_TOKEN }}
-          body: ${{ steps.changelog_reader.outputs.log_entry }}
+          body: ${{ steps.changelog_reader.outputs.changes }}
```

Comment:
This would also need `uses: ncipollo/release-action@v2` I assume?

Reply:
> This would also need `uses: ncipollo/release-action@v2` I assume?

Mmh, I don't think so. Why should we need it?

Reply:
Oh I see, we are already using `mindsers/changelog-reader-action@v2`. Got those mixed up.